README.md

```diff
@@ -51,3 +53,31 @@ docker rm updater2-win
 
 ```
 Search for **updater2.exe** in build-docker directory.
+
+## Build Mac version
+
+### Build Qt
+```
+curl -LO https://download.qt.io/archive/qt/5.14/5.14.2/single/qt-everywhere-src-5.14.2.tar.xz
+tar -xzf qt-everywhere-src-5.14.2.tar.xz
+cd qt-everywhere-src-5.14.2
+MODULES=qtbase,qtquickcontrols,qtquickcontrols2,qtsvg,qtgraphicaleffects
+# Default install location (--prefix): /usr/local/Qt-5.14.2
+./configure -opensource -confirm-license -release -optimize-size -no-shared -no-framework -static --c++std=14 -nomake tests -nomake tools -nomake examples
+eval make -j(whatever) module-{$MODULES}
```

Comment:
$(sysctl -n hw.logicalcpu)

Reply:
Is `hw.logicalcpu` better than `sysctl -n hw.ncpu`, I guess the second one only list cores and while first one also list threads?

Reply:
ya, if you have hyperthreading.